tests/bsim/bluetooth/audio/src/bap_broadcast_source_test.c

```diff
@@ -611,11 +611,14 @@ static void test_main(void)
 	/* Update metadata while streaming */
 	test_broadcast_source_update_metadata(source, adv);
 
-	/* Wait for other devices to have received what they wanted */
-	backchannel_sync_wait_any();
-
-	/* Wait for other devices to let us know when we can stop the source */
-	backchannel_sync_wait_any();
+	/* wait for all acceptors to:
+	 * 1. have received data
+	 * 2. let us know that we can stop the source
+	 */
+	for (int i = 0; i < acceptor_cnt; i++) {
+		backchannel_sync_wait_any();
+		backchannel_sync_wait_any();
+	}
```

Comment:
`backchannel_sync_wait_all` can be used instead

Reply:
If switching to `backchannel_sync_wait_all`, you may be able to revert all the other changes except adding the additional acceptor